packages/aws-cdk-lib/aws-dynamodb/test/table-v2.test.ts

```diff
@@ -1293,6 +1299,72 @@ describe('replica tables', () => {
     });
   });
 
+  test('with TagAspect', () => {
+    // GIVEN
+    const stack = new Stack(undefined, 'Stack', { env: { region: 'us-east-1' } });
+
+    // WHEN
+    const table = new TableV2(stack, 'Table', {
+      partitionKey: { name: 'pk', type: AttributeType.STRING },
+      replicas: [{
+        region: 'us-west-1',
+      }],
+    });
+
+    Tags.of(table).add('tagKey', 'tagValue');
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::GlobalTable', {
+      Replicas: [
+        {
+          Region: 'us-west-1',
+          Tags: [{ Key: 'tagKey', Value: 'tagValue' }],
+        },
+        {
+          Region: 'us-east-1',
+          Tags: [{ Key: 'tagKey', Value: 'tagValue' }],
+        },
+      ],
+    });
+  });
+
+  test('replica tags override global table tags', () => {
+    // GIVEN
+    const stack = new Stack(undefined, 'Stack', { env: { region: 'us-east-1' } });
+
+    // WHEN
+    const table = new TableV2(stack, 'Table', {
+      partitionKey: { name: 'pk', type: AttributeType.STRING },
+      replicas: [{
+        region: 'us-west-1',
+        tags: [{ key: 'tableTagProperty', value: 'replicaTagPropertyValue' }],
+      }],
+      tags: [{ key: 'tableTagProperty', value: 'globalTagPropertyValue' }],
+    });
+
+    Tags.of(table).add('tagAspect', 'tagAspectValue');
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::DynamoDB::GlobalTable', {
+      Replicas: [
+        {
+          Region: 'us-west-1',
+          Tags: [
+            { Key: 'tableTagProperty', Value: 'replicaTagPropertyValue' },
+            { Key: 'tagAspect', Value: 'tagAspectValue' },
+          ],
+        },
+        {
+          Region: 'us-east-1',
+          Tags: [
+            { Key: 'tableTagProperty', Value: 'globalTagPropertyValue' },
+            { Key: 'tagAspect', Value: 'tagAspectValue' },
+          ],
```

Comment:
Is this the right behaviour? `tableTagProperty` is only added to `us-west-1` replica but showed up in `us-east-1` too.

Reply:
Our current documentation seems to specifically say `You can add tags to a `TableV2` in several ways. By adding the tags to the construct itself it will apply the tags to the primary table.`

Reply:
https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-dynamodb/README.md#tags